extensionsGallery-insider.json

```diff
@@ -5511,6 +5511,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "109",
+					"extensionName": "couchbase",
+					"displayName": "Couchbase",
+					"shortDescription": "Welcome to the official Visual Studio Code extension for Couchbase! This extension is designed to provide a seamless experience for Couchbase Server users who want to work within the popular Visual Studio Code editor.",
```

Comment:
Update the description to mention Azure Data Studio instead of Visual Studio Code since those are two different products.